homeassistant/components/hassio/__init__.py

```diff
@@ -124,6 +138,17 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+get_supervisor_ip = deprecated_function(
+    "homeassistant.helpers.hassio.get_supervisor_ip", breaks_in_ha_version="2025.11"
+)(_get_supervisor_ip)
+is_hassio = deprecated_function(
```

Comment:
Will this work when called via `hass.components.hassio.is_hassio()`?

The `bind_hass` decorator is deprecated but not removed yet.

Reply:
Now the function has again `@bind_hass`